base85/base85.go

```diff
@@ -0,0 +1,76 @@
+package base85
```

Comment:
I will not give you extra score for this solution, because this is Ilya's solution. Please implement your own original solution.

In the future I won't score any non-original solutions.